src/core/EdFi.DataManagementService.Core/DmsCoreServiceExtensions.cs

```diff
@@ -0,0 +1,34 @@
+// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+using EdFi.DataManagementService.Core.ApiSchema;
+using EdFi.DataManagementService.Core.Backend;
+using EdFi.DataManagementService.Core.Validation;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace EdFi.DataManagementService.Core;
+
+/// <summary>
+/// The DMS service extensions to be registered to a Frontend DI container
+/// </summary>
+public static class DmsCoreServiceExtensions
+{
+    /// <summary>
+    /// The DMS default service configuration
+    /// </summary>
+    public static IServiceCollection AddDmsDefaultConfiguration(this IServiceCollection services)
+    {
```

Comment:
Move all the container config out of AspNetCore. Instead of exposing all these internal classes and interfaces, just expose a default DMS Core configuration.